projects/kit/components/input-count/input-count.template.html

```diff
@@ -35,7 +35,7 @@
             class="button button_plus"
             [icon]="iconUp"
             [appearance]="options.appearance"
-            [title]="texts[0]"
+            [title]="$any(texts)[0]"
```

Comment:
Why again and below?

Reply:
<img width="622" alt="image" src="https://user-images.githubusercontent.com/12021443/156890018-bcd13c2a-b54a-43bb-b106-b6eec5a75f31.png">

because `text` type is `[string, string] | null`

Reply:
fixed